pkg/kubectl/describe.go

```diff
@@ -173,13 +173,24 @@ func (d *NamespaceDescriber) Describe(namespace, name string, describerSettings
 	}
 	resourceQuotaList, err := d.Core().ResourceQuotas(name).List(api.ListOptions{})
 	if err != nil {
-		return "", err
+		if errors.IsNotFound(err) {
```

Comment:
I wanted to put a very specific fix to minimize the risk of getting this cherrypicked in 1.4.
I will prefer doing a generic fix in another PR, unless you really really want me to do it in this PR.